src/tests/functional/shared_test_classes/include/shared_test_classes/subgraph/broadcast_eltwise_eliminated.hpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "shared_test_classes/base/ov_subgraph.hpp"
+
+namespace ov {
+namespace test {
+
+class BroadcastEltwiseEliminated : public testing::WithParamInterface<const char*>,
+                                    virtual public ov::test::SubgraphBaseTest {
```

Comment:
do we really need this "virtual" inheritance here?

Reply:
seems like we don't